nix/shells.nix

```diff
@@ -21,7 +21,7 @@
 
           pkgs.just
           pkgs.cargo-nextest
-
+          pkgs.bun
```

Comment:
I don't believe bun is necessary given that `pnpm` and `nodejs` is a dependency in `doc/package.nix`